docs/docs/guides/labs/dg/deploying-dg.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Deploying to Dagster+'
+sidebar_position: 350
+---
+
+import DgComponentsPreview from '@site/docs/partials/\_DgComponentsPreview.md';
+
+<DgComponentsPreview />
+
+This guide covers how to deploy a `dg`-compatible project to Dagster+.
+
+## Prerequisites
+
+- The `dg` command-line interface should be [installed](/guides/labs/dg#installation).
+- A `dg`-compatible project. If you don't have one, you can create one by either:
+  - [Initializing a new project](/guides/labs/dg/scaffolding-a-project)
+  - [Migrating an existing project to dg](/guides/labs/dg/incrementally-adopting-dg/migrating-project)
+- A [Dagster+ account](https://dagster.cloud/signup).
+- If you're using Dagster+ Hybrid, a [deployed Hybrid agent in your cloud environment](dagster-plus/deployment/deployment-types/hybrid),
+with access to a Docker container registry to which you can push images.
+
+
+## Deploying to Dagster+ from your local machine
+
+### 1. Set up authentication to your deployment with the `dg` CLI:
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/deploying-dg/1-dg-plus-login.txt" />
+
+### 2. Ensure your `dg` project includes the `dagster-cloud` package in the dependencies in its `pyproject.toml` file.
+
+If you're using `uv`, you can run:
```

Comment:
Should we have tabbed uv/pip steps here?

Reply:
I don't think there's a corresponding pip command for this one

Reply:
(But we could say if you're not, manually add it to your project file)